app/views/nsm/steps/claim_details/edit.html.erb

```diff
@@ -6,9 +6,9 @@
     <%= govuk_error_summary(@form_object) %>
     <h1 class="govuk-heading-xl"><%= t('.heading') %></h1>
     <%= step_form @form_object do |f| %>
-      <%= f.govuk_number_field :prosecution_evidence,label:{ size: 's'}, min: 1, step: 1, width: 3 %>
-      <%= f.govuk_number_field :defence_statement,label:{ size: 's'}, min: 1, step: 1, width: 3 %>
-      <%= f.govuk_number_field :number_of_witnesses,label:{ size: 's'}, min: 1, step: 1, width: 3 %>
+      <%= f.govuk_text_field :prosecution_evidence,label:{ size: 's'}, width: 3 %>
```

Comment:
Couple of things with these. First off, if we're using text fields for numbers we should include `inputmode=numeric` as this prompts mobile devices to show the right sort of keyboard.

Second, now that these fields do accept non-numeric inputs, the request from Mary in PA was that if a user enters a non-numeric value and submits the form, the thing they originally entered should be pre-populated in the field along with a validation error message. As it stands this won't happen, because all these attributes are marked as `:integer` in the form classes, meaning they'll get parsed to integers before validation even happens, so will be converted to `0`, which may not even trigger a validation error, and may not be what the provider intended to enter.

The way around this one is to change the attribute type from `:integer` to `:fully_validatable_integer`. But note that when you do that, you need to validate `is_a_number: true` and add a `not_a_number` error string.

Finally, it'd be worth checking the existing validations on all these fields. E.g. the message for number_of_hearing if you put a 0 in is `must be greater than 0`, i.e. there's no explicit error message defined so Rails is making up its own, bad,  one.